.github/workflows/actions.yml

```diff
@@ -389,6 +389,7 @@ jobs:
       - name: Install
         run: cmake --build obj --config RelWithDebInfo --target install
       - name: Package
+        if: ${{ needs.what-to-make.outputs.make-dist == 'true' || needs.what-to-make.outputs.make-qt == 'true' }}
```

Comment:
> Another option would be to always enable daemon and Qt client builds, so that MSI package could always be built (and available for testing, if needed).

This afternoon I'm trying to focus on Fixing The Things That Make CI Go Red and that feels like scope creep.

But always enabling daemon + Qt sounds reasonable. That would be worth coming back to, or worth doing it sooner if you want to work on it :slightly_smiling_face: